engine/src/main/scala/cromwell/engine/workflow/lifecycle/OutputsLocationHelper.scala

```diff
@@ -0,0 +1,78 @@
+package cromwell.engine.workflow.lifecycle
+
+import cromwell.backend.{
+  AllBackendInitializationData,
+  BackendConfigurationDescriptor,
+  BackendInitializationData,
+  BackendLifecycleActorFactory
+}
+import cromwell.core.CallOutputs
+import cromwell.core.WorkflowOptions.UseRelativeOutputPaths
+import cromwell.core.path.{Path, PathCopier, PathFactory}
+import cromwell.engine.EngineWorkflowDescriptor
+import cromwell.engine.backend.{BackendConfiguration, CromwellBackends}
+import wom.values.{WomSingleFile, WomValue}
+
+trait OutputsLocationHelper extends PathFactory {
```

Comment:
Will be brought in to `WorkflowExecutionActor` for `pushWorkflowOutputMetadata` and `WorkflowCallbackActor` for `CallbackMessage`.